packages/dds/merge-tree/src/test/properties.spec.ts

```diff
@@ -38,11 +38,11 @@ describe("Properties", () => {
 		it("empty properties match", () => {
 			assert(matchProperties({}, {}));
 		});
-		it("undefined and empty properties don't match", () => {
-			assert(!matchProperties(undefined, {}));
+		it("undefined and empty properties match", () => {
+			assert(matchProperties(undefined, {}));
 		});
-		it("empty and undefined properties don't match", () => {
-			assert(!matchProperties({}, undefined));
+		it("empty and undefined properties match", () => {
+			assert(matchProperties({}, undefined));
 		});
```

Comment:
Add tests for undefined != {a:"a"} and {} != {a:"a"}. Weird that these cases aren't already covered.